model/src/w3wavemd.F90

```diff
@@ -1581,12 +1581,12 @@ SUBROUTINE W3WAVE ( IMOD, ODAT, TEND, STAMP, NO_OUT &
               FLUSH(740+IAPROC)
 #endif
 #endif
```

Comment:
@aronroland while not strictly related to this PR, I was looking into the commented out code below and I think the '#endif` on line 1583 does not have a matching '#ifdef` I think basically from line 1470 to line 1592 (line numbers from your branch) should basically all be under a '#ifdef W3_PDLIB' and am curious if you agree?  As this is technically not part of this PR, I'm happy to raise this as a separate bug-fix but I did find it because of the review of this PR and am commenting here on the off chance this caused problems. I haven't tested this theory yet by say removing an ifdef and recompiling and running regtests on the code, but if you agree I can start that testing first thing  in the morning.

Reply:
well well, it would not compile but i was now looking at it (quite late now) and it is not nicely done so I will clean the whole ifdefs to make it more clean. 

Many thanks for pointing this out. I will check the code tomorrow again carefully and commit my changes. I would not make a new bug-fix since it is not a bug otherwise the code would not compile but it is not nice nor well readable, which is not acceptable.

Reply:
hello @JessicaMeixner-NOAA ok i cleaned it, compiled and pushed it please check it. If u disagree or u it shows up that it is a bug feel free to revert and lets catch up tomorrow on this issue. I also find some more comments that can be cleaned.

Reply:
Thanks @aronroland  I'm watching the regtests go and will check if I see anything, but i agree with you this is much easier to read.  I don't think what you have here is a bug, but will run tests to confirm.